projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -125,12 +120,7 @@
         </ng-container>
       </ng-container>
       <ng-template #title>
-        <span
-          role="heading"
-          aria-level="4"
-          *ngIf="node.title"
-          [attr.tabindex]="-1"
-        >
+        <span role="presentation" *ngIf="node.title" [attr.tabindex]="0">
```

Comment:
This text is used as a heading for the following elements and should be recognized as one. Giving it a role of `presentation` would hide it from a11y software, which is not allowed for elements that convey information visible on the screen.

Elements that cannot be interacted with directly should not have tabindex of 0